prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -528,11 +529,20 @@ class LookupTableCreateCommand : GenericLookupTableCommand(
     private val tableName by option("-n", "--name", help = "The name of the table to perform the operation on")
         .required()
 
+    /**
+     * The table Sha256 checksum.
+     */
+    private val tableSha256 by option("-c", "--checksum", help = "The Sha256 checksum of the table")
```

Comment:
Don't let a user specify a checksum.  You need to compute it every time.

Reply:
Removed this variable.